README.md

```diff
@@ -57,24 +57,24 @@ By default, this package runs using your destination and the `recharge` schema.
 
 ```yml
 vars:
-    recharge_database: your_destination_name
-    recharge_schema: your_schema_name 
+  recharge_database: your_destination_name
+  recharge_schema: your_schema_name 
 ```
 
-## Step 4: Disable models for non-existent sources
-Your Recharge connector may not sync every table that this package expects. If you do not have the `PAYMENT_METHOD` and/or `ONE_TIME_PRODUCT` tables synced, add the corresponding variable(s) to your root `dbt_project.yml` file to disable these sources:
-
-```yml
-vars:
-  recharge__payment_method_enabled: false # Disables if you do not have the PAYMENT_METHOD table. Default is True.
-  recharge__one_time_product_enabled: false # Disables if you do not have the ONE_TIME_PRODUCT table. Default is True.
+## Step 4: Disable models for non-existent sources 
+Your Recharge connector may not sync every table that this package expects. If you do not have the `PAYMENT_METHOD` and/or `ONE_TIME_PRODUCT` tables synced, add the corresponding variable(s) to your root `dbt_project.yml` file to disable these sources: 
+ 
+```yml 
+vars: 
+  recharge__payment_method_enabled: false # Disables if you do not have the PAYMENT_METHOD table. Default is True. 
+  recharge__one_time_product_enabled: false # Disables if you do not have the ONE_TIME_PRODUCT table. Default is True. 
```

Comment:
I noticed there is no documentation here for the `recharge__charge_tax_line_enabled` variable. Would you be able to add that in here. 

Also don't forget to add this to the source package as well.